static/js/alert_words_ui.js

```diff
@@ -10,9 +10,6 @@ import * as ListWidget from "./list_widget";
 export let loaded = false;
 
 export function render_alert_words_ui() {
-    if (!loaded) {
-        return;
-    }
```

Comment:
Why are you removing this?

Reply:
adding this would need me to add a test to check wh

> The first and third commit should be squashed as we always have tests in the same commit in which the original code is added. Now, running tests on first commit will fail for coverage as the tests for it are in the third commit and we target that tests should be passed for each individual commit.

got it, updated the PR.

thanks :)

Reply:
Adding this would then need me to add test for making it pass in --coverage!

And for which i added 
```
    assert.equal(alert_words_ui.loaded, false);
    assert.equal(alert_words_ui.render_alert_words_ui(),undefined)
```
and you asked for avoiding to return `undefined` in tests .

Reply:
so, would it be OK if i add this line of returning `undefined` ?

Reply:
I just said for testing this we can check whether `list_widget_create_called` is true/false, to make sure we return early when loaded is false.

Reply:
okay, got the point, so i can add these lines in the tests right ?

```
    assert.equal(alert_words_ui.loaded, false);
    assert.equal(alert_words_ui.render_alert_words_ui(),undefined)

```

Reply:
Updated the PR.